doc/doxygen/headers/boundary.h

```diff
@@ -79,64 +79,53 @@
  * <tt>boundary_id</tt> that uniquely identifies which part of the
  * boundary this face is on. If nothing is specified at creation time,
  * each boundary face has a zero boundary id and each triangulation
- * object has an invalid manifold id. On the other hand, the boundary
+ * object has an flat manifold id. On the other hand, the boundary
```

Comment:
I guess that, to be consistent with the majority of the documentation below, this should be `manifold_id`.

(sorry for the duplicate indications later on)